test/3nodetest/Makefile

```diff
@@ -14,7 +14,7 @@ data/filetiny: Makefile
 	cp Makefile ./data/filetiny # simple
 
 data/filerand: bin/random
-	./bin/random 50000000 > ./data/filerand
+	./bin/random 50MB > ./data/filerand
```

Comment:
NB: when we parallelize this test, we'll want to create different data dirs per run. we'd still want it local so a simple `make clean` blows them all out